BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -499,11 +499,21 @@ Cryptographic algorithms weaken over time as cryptanalysis improves: algorithms
                with the string "TLS_RSA_WITH_*", have several drawbacks, especially
                the fact that they do not support forward secrecy.
             
+* Implementations SHOULD NOT negotiate cipher suites based on
+               non-ephemeral (static) finite-field Diffie-Hellman key agreement.
+
+  Rationale: These cipher suites, which have assigned values prefixed by "TLS_DH_*", have several drawbacks, especially
+               the fact that they do not support forward secrecy.
 
 * Implementations MUST support and prefer to negotiate cipher suites 
                offering forward secrecy, such as those in the Ephemeral 
                Diffie-Hellman and Elliptic Curve Ephemeral Diffie-Hellman ("DHE" 
-               and "ECDHE") families.
+               and "ECDHE") families.  However, given the known fragility
+               of the construction (see {{RACCOON}}) and the limitation around
+               negotiation (including using {{?RFC7919}}, which has seen very
+               limited uptake), TLS 1.2 implementations SHOULD NOT negotiate
+               cipher suites based on ephemeral finite-field Diffie-Hellman key
+               agreement (i.e., "TLS_DHE_*" suites).
```

Comment:
This bullet has become confusing. I suggest to reword and remove DHE from the MUST sentence at the top.

Reply:
Done in 66c60e0